### PR TITLE
Issue 146 - make output image optional

### PR DESCRIPTION
**Pull request by soltysh**
@bparees as promised, PTAL

### CODE REVIEW

**Comment by soltysh** [comment]
[test]

File: pkg/build/strategies/sti/sti.go [hunk 1]
```diff
@@ -216,22 +216,29 @@ func (b *STI) PostExecute(containerID string, location string) error {
...
-		Command:     append(cmd, filepath.Join(location, api.Run)),
+		Command:     append([]string{}, filepath.Join(location, api.Run)),
 		Env:         buildEnv,
 		ContainerID: containerID,
-		Repository:  b.request.Tag,
 	}
+	if len(b.request.Tag) > 0 {
+		opts.Repository = b.request.Tag
+	}
+
```
Comment (csrwng) [inline pkg/build/strategies/sti/sti.go]: Do you really need this change? Wouldn't opts.Repository remain blank if you just assign it b.request.Tag (if blank)?

Comment (bparees) [inline pkg/build/strategies/sti/sti.go]: +1

**Comment by bparees** [comment]
i second @csrwng 's comment, otherwise lgtm.

**Comment by soltysh** [comment]
Fixed, now waiting for the tests to pass and I'm going to merge it.

**Comment by soltysh** [comment]
[merge]

**Pull request merged**